### DISCUSSION TITLE
When can we expect a new Beets release ?

### DISCUSSION THREAD

**Discussion by doctorfree** [beetbox/beets#4430] [General]
Beets 1.6.0 was released last November. It's 8 months since that release. I am working on a new release of [MusicPlayerPlus](https://github.com/doctorfree/MusicPlayerPlus#readme) which incorporates support for Beets integration. I've developed, tested, and documented this release with Beets 1.6.0. However, I would prefer to postpone release until Beets 1.6.1 or later is available if that is planned soon-ish.

Do we have an expected timeframe for a new release of Beets?

Thanks for any info even if that info is simply "we do not have an expected timeframe for a new release".

**Comment by sampsyo** [comment]
I would like to take care of this as soon as I have a moment to wrap up some loose ends.

**Reply by doctorfree** [reply]
Thanks for the prompt reply and info. I do not want to contribute to any rush or hurried release. That is, please take as much time as you need. I know how carefully releases must be planned, tested, staged, and deployed. Updates can be difficult especially when preserving compatibility.

Beets is great. Thanks!

**Comment by wisp3rwind** [comment]
Hi @sampsyo, I just wanted to point out that I'd consider the limit rework of #4362 to be one of the loose end to be tied up before a release, since we're probably going to remove the `lslimit` plugin (which has not seen a release yet) in favor of adding the functionality in beets' core.

**Reply by sampsyo** [reply]
That's a good point.
